plugin/src/main/groovy/com/btkelly/gnag/reporters/AndroidLintViolationDetector.groovy

```diff
@@ -82,7 +87,7 @@ class AndroidLintViolationDetector extends BaseViolationDetector {
                         violationType,
                         name(),
                         nullableCommentInHtml,
-                        computeFilePathRelativeToProjectRoot((String) violation.location.@file.text()),
+                        PathCalculator.calculatePathWithinProject(project, (String) violation.location.@file.text()),
```

Comment:
Refactored to allow Checkstyle parsing code (now shared by Checkstyle and ktlint checkers) to call these methods from outside the `reporters` package.